services/keyword_service/keyword.go

```diff
@@ -107,9 +108,17 @@ func (k *KeywordService) ReadFile(filename string) ([]string, error) {
 
 func (k *KeywordService) UploadFile(c *gin.Context, file *multipart.FileHeader) string {
 	path := "dist/"
-	_ = os.Mkdir(path, 0755)
+	err := os.Mkdir(path, 0755)
+	if err != nil {
+		log.Warn("Failed to create directory: ", err)
```

Comment:
@gutakk Why using `warn` instead of `error` here? Will the code below still works if the directory is not created?

Reply:
Good catch. Fixed.